src/tests/Microsoft.VisualStudio.Composition.Tests/XunitExtensions/AssertFailedException.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Microsoft.VisualStudio.Composition.Tests
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    [Serializable]
+    internal class AssertFailedException : Exception
+    {
+        public AssertFailedException()
```

Comment:
Fixed. But hopefully you're aware that this practice makes it harder to switch to public in the future (instead of fixing the modifier on the type you also need to fix on all members). I'm guessing this is what you want though.